### DISCUSSION TITLE
Messaging: SENDER_ID_MISSMATCH

### DISCUSSION THREAD

**Discussion by dlindstrm** [invertase/react-native-firebase#5433] [Q&A]
We use the messaging package to get the fcm token, then token then is stored in firestore. When sending pushnotifications from our backend to each one of these tokens, approximately 1% throws a `SENDER_ID_MISMATCH` error. I have no clue why this is happening. The clients all write to the correct firestore, any idea how this can occur?

```
curl --request GET 'https://iid.googleapis.com/iid/info/<firebase token>' \
--header 'Authorization: Bearer <server key>'
```
When I run the snippet above, all tokens that throws `SENDER_ID_MISMATCH` when sending a notification returns a 403, so there is clearly something wrong.

**Comment by mikehardy** [comment]
This appears to be some sort of configuration error somehow. I believe it's a general (that is: not react-native-firebase specific) firebase usage question so you may have more luck on stackoverflow, similar to the reasonable-looking answers I scanned here https://duckduckgo.com/?q=firebase+messaging+sender_id_mismatch
